### PR TITLE
rewrite-observability-platform-introduction page

### PR DESCRIPTION
**Pull request by QuentinBisson**
### What this PR does / why we need it

Towards https://github.com/giantswarm/giantswarm/issues/28925

This page is quite outdated.

We also deprecated EFK stack a while back. Should I remove the section about this app? 

Because it is weird that we have this at the end:

image

### Things to check/remember before submitting

- If you made content changes

    - Run `make dev` to render and proofread content changes locally.
    - Bump `last_review_date` in the front matter header if you reviewed the entire page.

### CODE REVIEW

File: src/content/vintage/platform-overview/observability/_index.md [hunk 1]
```diff
@@ -17,33 +17,38 @@ user_questions:
   - What app do you recommend for monitoring?
   - What app do you recommend for logging?
   - What app do you recommend for tracing?
-last_review_date: 2022-11-18
+last_review_date: 2024-02-28
 ---
 
-Observability is based on three main data sources: logs, metrics and tracing\*. To cover these needs Giant Swarm provides its customers with a managed observability stack. The apps included in this stack are listed below. We chose these apps based on our general principles of preferring open source over proprietary solutions, as well as simplicity, performance, and speed over complexity.
+Observability is based on four main data sources: __logs__, __metrics__, __traces__ and __profiles__. To cover these needs Giant Swarm provides its customers with a fully managed observability platform provided with 24//7 monitoring and alerting The tools included in the observabily platform are listed below.
```
Comment (stone-z) [inline src/content/vintage/platform-overview/observability/_index.md:23]:
```suggestion
Observability is based on four main data sources: __logs__, __metrics__, __traces__ and __profiles__. To cover these needs, Giant Swarm provides its customers with a fully managed observability platform supported by 24//7 monitoring and alerting The tools included in the observabily platform are listed below.
```

File: src/content/vintage/platform-overview/observability/_index.md [hunk 2]
```diff
@@ -17,33 +17,38 @@ user_questions:
...
-In brief, super fast response times from engineers, 24//7 monitoring and alerting by us, and in general, having shared goals and responsibilities to keep services up and get them up when they go down.
+If you want to learn more about observability, we recommend that you read [this](https://opentelemetry.io/docs/concepts/observability-primer/)
 
-In addition, we strive to uphold our "30 Day Upgrade Promise," our commitment to keep all managed apps within 30 days of latest upstream releases.
+To summarize, for observability, we recommend and provide these tools:
 
-To summarize, for observability, we recommend and provide these four managed apps:
+## Loki for logging
+
+To store, index and make logs (containers, kubernetes events, audit logs) searchable. You can also install our [Loki App](https://github.com/giantswarm/loki-app/blob/master/README.md) on your clusters.
```
Comment (stone-z) [inline src/content/vintage/platform-overview/observability/_index.md:33]:
```suggestion
To store, index, and make logs (containers, kubernetes events, audit logs) searchable. You can also install our [Loki App](https://github.com/giantswarm/loki-app/blob/master/README.md) on your clusters.
```

Comment (stone-z) [reply src/content/vintage/platform-overview/observability/_index.md:33]: IIUC, the implied distinction here is that we run Loki on MCs for logs we collect, but customers must also install Loki themselves if they want to use it for their own log collection. Is that right?
If so, that could be more explicit here as I found it slightly confusing

**Review by stone-z — APPROVED** [review]
Couple nits but LGTM.

File: src/content/vintage/platform-overview/observability/_index.md (hunk 2 above)
Comment (QuentinBisson) [reply src/content/vintage/platform-overview/observability/_index.md]: Well the implied direction is that we will bé able to store their logs if they want to but they  can also install it by themselves if they want to, but maybe that part should be explained better. I'm not sure how though 😂

Comment (stone-z) [reply src/content/vintage/platform-overview/observability/_index.md]: What is the benefit if they run Loki themselves?

File: src/content/vintage/platform-overview/observability/_index.md [hunk 3]
```diff
@@ -17,33 +17,38 @@ user_questions:
   - What app do you recommend for monitoring?
   - What app do you recommend for logging?
   - What app do you recommend for tracing?
-last_review_date: 2022-11-18
+last_review_date: 2024-02-28
 ---
 
-Observability is based on three main data sources: logs, metrics and tracing\*. To cover these needs Giant Swarm provides its customers with a managed observability stack. The apps included in this stack are listed below. We chose these apps based on our general principles of preferring open source over proprietary solutions, as well as simplicity, performance, and speed over complexity.
+Observability is based on four main data sources: __logs__, __metrics__, __traces__ and __profiles__. To cover these needs, Giant Swarm provides its customers with a fully managed observability platform supported by 24//7 monitoring and alerting The tools included in the observabily platform are listed below.
```
Comment (QuantumEnigmaa) [inline src/content/vintage/platform-overview/observability/_index.md:23]:
```suggestion
Observability is based on four main data sources: __logs__, __metrics__, __traces__ and __profiles__. To cover these needs, Giant Swarm provides its customers with a fully managed observability platform supported by 24//7 monitoring and alerting. The tools included in the observabily platform are listed below.
```

File: src/content/vintage/platform-overview/observability/_index.md [hunk 4]
```diff
@@ -17,33 +17,38 @@ user_questions:
...
 
-## Grafana as data visualization tool
+We are currently offering Grafana Tempo a part of our observability platform on demand and without any availability garantees due to the low priority it has with our customers.
 
-To analyze, visualize, and correlate metrics from Prometheus and Linkerd as well as logs from Loki.
+## Pyroscope for continuous profiling
 
-These pieces can be separately or together. Adopting them together and as fully managed apps from Giant Swarm gives you the benefit of using vetted open source tools that you know work together to provide you with consistency and coverage.
+Profiling is quite a new kid on the block when it comes to observability.
+Same as with Grafana Tempo, we are currently offering Grafana Tempo a part of our observability platform on demand and without any availability garantees due to the low priority it has with our customers.
```
Comment (QuantumEnigmaa) [inline src/content/vintage/platform-overview/observability/_index.md:50]:
```suggestion
Same as with Grafana Tempo, we are currently offering Grafana Pyroscope a part of our observability platform on demand and without any availability garantees due to the low priority it has with our customers.
```

**Review by QuantumEnigmaa — APPROVED** [review]
LGTM

**Review by pipo02mix — APPROVED** [review]
LGTM though in new structure it may find a different place in the hierarchy. Elastic can be removed yeah :)
